script.js

```diff
@@ -252,9 +268,87 @@ function isMoving(ghost)
     }
 }
 
-function distance(player, ghost)
-{
-    return distanceBetween(player, ghost)
+// function distance(player, ghost)
+// {
+//     return distanceBetween(player, ghost)
+// }
+
+
+
+
+
+
+// Ghost functions:
+
+
+// Helper function to return a specific ghost
+function getGhost(color) {
+    const GHOSTS = ['pink', 'red', 'blue', 'yellow'];
+    for (let i = 0; i < GHOSTS.length; i++)
+        if (GHOSTS[i] == color) return ghosts.children.entries[i];
```

Comment:
We could use [Array.find()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) here